### PR TITLE
feat(EMS-510): Policy and exports - Single contract policy - policy currency code validation

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR adds "contract currency code" field validation to the Single contract policy page and ensures that a previously submitted value is displayed in the form. 

## Summary of changes

- Add validation rule to check that the field is not empty.
- Add policy currency code to the graphQL mutation.
- Add tests.
- Update "save and go back" unit test to ensure the functionality works when the form has every valid field provided.

### CODE REVIEW

File: e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/single-contract-policy/single-contract-policy.spec.js [hunk 1]
```diff
@@ -306,6 +306,7 @@ context('Insurance - Policy and exports - Single contract policy page - As an ex
 
         singleContractPolicyPage[TOTAL_CONTRACT_VALUE].input().should('have.value', '10000');
         singleContractPolicyPage[CREDIT_PERIOD_WITH_BUYER].input().should('have.value', 'mock free text');
+        singleContractPolicyPage[POLICY_CURRENCY_CODE].inputOptionSelected().contains('GBP');
```
Comment (Zainzzkk) [inline e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/single-contract-policy/single-contract-policy.spec.js:309]: can we make currencies constants like in DTFS?

File: e2e-tests/cypress/support/insurance/complete-and-submit-single-contract-policy-form.js [hunk 2]
```diff
@@ -34,6 +35,7 @@ export default () => {
 
   singleContractPolicyPage[TOTAL_CONTRACT_VALUE].input().type('10000');
   singleContractPolicyPage[CREDIT_PERIOD_WITH_BUYER].input().type('mock free text');
+  singleContractPolicyPage[POLICY_CURRENCY_CODE].input().select('GBP');
```
Comment (Zainzzkk) [inline e2e-tests/cypress/support/insurance/complete-and-submit-single-contract-policy-form.js:38]: can this be a constant?

**Comment by ttbarnes** [comment]
Great shout @Zainzzkk Can we do this in a future ticket please? There are e.g 61 files with "GBP". I can create a tech ticket to do this as a top priority tech improvement. Just don't want this PR to become huge, especially whilst we have a PR backlog.

**Review by Zainzzkk — APPROVED** [review]
LGTM

**Comment by ttbarnes** [comment]
Thanks @Zainzzkk ticket created: https://ukef-dtfs.atlassian.net/browse/EMS-787

**Pull request merged**